auth/src/main/java/io/grpc/auth/GoogleAuthLibraryCallCredentials.java

```diff
@@ -77,6 +90,14 @@ public void thisUsesUnstableApi() {}
   @Override
   public void applyRequestMetadata(MethodDescriptor<?, ?> method, Attributes attrs,
       Executor appExecutor, final MetadataApplier applier) {
+    SecurityLevel security = checkNotNull(attrs.get(ATTR_SECURITY_LEVEL), "securityLevel");
```

Comment:
Is this guaranteed to be set?

Reply:
API says so, although it depends on transport and channel implementation, and we already had a case internally where a custom channel is not setting it. Probably safer to tolerate null here.